.pylintrc

```diff
@@ -6,4 +6,10 @@ score=no
 
 [MESSAGES CONTROL]
 disable=all
-enable=unreachable
\ No newline at end of file
+enable=
+    assert-on-tuple,
```

Comment:
Does this even work?

```bash
find mesonbuild -name "*.py" -type f -exec grep 'assert(' {} \;
```

prints a ton of results.

Reply:
You probably need to feel for `assert(.*,` to find what it's actually searching for

Reply:
Oh, right. I was thinking of the `superfluous-paren` warnings, nevermind.